src/main/java/gov/cancer/pageobject/crosscutting/HttpProtocolPage.java

```diff
@@ -0,0 +1,47 @@
+package gov.cancer.pageobject.crosscutting;
+
+
+import gov.cancer.framework.Configuration;
+import io.restassured.response.*;
+
+import static io.restassured.RestAssured.when;
+
+/**
+ * This is a pseudo Page Class that is used solely for sending API requests
+ * RestAssured library is used for sending a request
+ *
+ * Currently it supports 'GET' request only, but can be extended to support POST requests,
+ * modifying headers, sending requests with provided JSON etc.
+ */
+public class HttpProtocolPage {
```

Comment:
This seems like an opportunity for an abstract `HttpPage` with an `HttpGetPage` as a subclass.

It's fine for now, but something to consider if/when we need to add other verbs.